### PR TITLE
POL-649 Remove old engine integration

### PR DESCRIPTION
**Pull request by gwenneg**
The old engine no longer exists. This PR removes all the integration code that was targeting it.

### CODE REVIEW

File: src/test/java/com/redhat/cloud/policies/app/PolicyCrudServiceTest.java [hunk 1]
```diff
@@ -65,44 +60,16 @@ void test() {
         assertEquals(5, history.getJsonObject("meta").getInteger("count"));
     }
 
-    private UUID createPolicy() {
+    @Transactional
```
Comment (bennetelli) [inline src/test/java/com/redhat/cloud/policies/app/PolicyCrudServiceTest.java:63]: why is this needed now?

Comment (gwenneg) [reply src/test/java/com/redhat/cloud/policies/app/PolicyCrudServiceTest.java:63]: Because of the SQL query that is executed in the new method body. The old version didn't execute a SQL query.

**Review by josejulio — APPROVED** [review]
lgtm
